doc/specs/stdlib_string_type.md

```diff
@@ -0,0 +1,1607 @@
+- `rhs`: Either scalar character value or string type. This argument is `intent(in)`.
+
+#### Result value
+
+The result is a default logical scalar value.
+
+#### Example
+
+```fortran
+use stdlib_string_type
+implicit none
+type(string_type) :: string
+logical :: res
+
+string = "bcd"
+res = llt(string, "abc")
+! res .eqv. .false.
+
+res = llt(string, "bcd")
+! res .eqv. .false.
+
+res = llt(string, "cde")
+! res .eqv. .true.
+end
+```
+
+
+<!-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -->
+### Lge function (lexical greater or equal)
+
+Lexically compare the order of two character sequences being greater or equal.
+
+#### Description
+
+The left-hand side, the right-hand side or both character sequences can
+be represented by a string type.
+This defines three procedures overloading the intrinsic `lge` procedure.
+
+#### Syntax
+
+`res = [[stdlib_string_type(module):lge(interface)]] (lhs, rhs)`
+
+#### Status
+
+Experimental
+
+#### Class
+
+Elemental function.
+
+#### Argument
+
+- `lhs`: Either scalar character value or string type. This argument is `intent(in)`.
+- `rhs`: Either scalar character value or string type. This argument is `intent(in)`.
+
+#### Result value
+
+The result is a default logical scalar value.
+
+#### Example
+
+```fortran
+use stdlib_string_type
+implicit none
+type(string_type) :: string
+logical :: res
+
+string = "bcd"
+res = lge(string, "abc")
+! res .eqv. .true.
+
+res = lge(string, "bcd")
+! res .eqv. .true.
+
+res = lge(string, "cde")
+! res .eqv. .false.
+end
+```
+
+
+<!-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -->
+### Lle function (lexical less or equal)
+
+Lexically compare the order of two character sequences being less or equal.
+
+#### Description
+
+The left-hand side, the right-hand side or both character sequences can
+be represented by a string type.
+This defines three procedures overloading the intrinsic `lle` procedure.
+
+#### Syntax
+
+`res = [[stdlib_string_type(module):lle(interface)]] (lhs, rhs)`
+
+#### Status
+
+Experimental
+
+#### Class
+
+Elemental function.
+
+#### Argument
+
+- `lhs`: Either scalar character value or string type. This argument is `intent(in)`.
+- `rhs`: Either scalar character value or string type. This argument is `intent(in)`.
+
+#### Result value
+
+The result is a default logical scalar value.
+
+#### Example
+
+```fortran
+use stdlib_string_type
+implicit none
+type(string_type) :: string
+logical :: res
+
+string = "bcd"
+res = lle(string, "abc")
+! res .eqv. .false.
+
+res = lle(string, "bcd")
+! res .eqv. .true.
+
+res = lle(string, "cde")
+! res .eqv. .true.
+end
+```
+
+
+<!-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -->
+### Comparison operator greater
+
+Compare the order of two character sequences being greater.
+
+#### Description
+
+The left-hand side, the right-hand side or both character sequences can
+be represented by a string type.
+This defines three procedures overloading the intrinsic `operator(.gt.)`.
```

Comment:
Related to the discussion during the monthly call, `.gt.` could become `>`

Reply:
Maybe the syntax section could display both the new and old forms for completeness?